CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.67.2] - 2021-09-14
+### Added
+- Added new Icon svg for Renewal-Line
+
```

Comment:
this looks wrong, new version should be 1.0.something - you might need to pull master and sort this out